src/DynamicNamespaces.tsx

```diff
@@ -28,7 +28,7 @@ export default function DynamicNamespaces({
 
   useEffect(() => {
     loadNamespaces()
-  }, [])
+  }, [namespaces])
```

Comment:
Here most people will put `namespaces={['dynamic']}`, causing a new array to be created in each render and therefore load in each render the namespace. Maybe it would make more sense to put `[namespaces.join()]` as a dependency so that it only changes if the string changes? What do you think about it? Thanks for your contribution @rihardssceredins.

Reply:
In our case the first namespace is for the specific page and the second one is for the common translations across all pages.

```js
namespaces={[
  `${landing.series}/quiz/${step}`,
  `${landing.series}/common`,
]}
```

So.. joining them would also work and would optimize the scenario you mentioned. :)
Pushed the changes!